tests/system/source/main.d

```diff
@@ -98,6 +98,8 @@ int main (string[] args)
                     const blocks = client.getBlocksFrom(height, 10);
                     writefln("%s %s has blocks:\n%s", PREFIX, nodeFromClientIndex(idx), prettify(blocks));
                     writefln("%s ----------------------------------------", PREFIX);
+                    assert(blocks[$ -1].txs.length > 0,
```

Comment:
```suggestion
                    assert(blocks[$-1].txs.length > 0,
```

Reply:
Actually, spaces around binary operators ;)

Reply:
I did a quick search and found [$-1] was the norm in our code.

Reply:
Although just found out that [$ - 1] is more common.